### PR TITLE
Add dark and light colors to ConfettiView animation

### PR DESCRIPTION
**Pull request by Gio2018**
Fixes #NA

This PR adds colors for both light and dark mode to the ConfettiView animation (see examples below)

light appearance | dark appearance
|:-:|:-:|
 | 

To test:

To test this PR, you'll either:
- need to point to a `sandbox` with your device or simulator, and have the backend diff for Milestone Notifications applied to it, 
- or in `Notification.swift`  edit the isViewMilestone property to match an existing notification type, e.g. a likes or "first post" milestone. In this case, in that property, you can replace note.type == "view_milestone" with note.type == "like_milestone_achievement" or with note.type == "post_milestone_achievement" (or both, if you want to test it on more notifications).
- you also need to enable the `milestoneNotifications` feature flag

Test steps:

- Build/run and go to notifications. Make sure you have some notifications of the types described in the pre-requisites
- Select a notification and make sure the confetti animation shows up and looks consistent with the examples above
- Perform the test in both light and dark appearance

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

cc: @mbshakti

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Notifications/Milestone Notifications/ConfettiView.swift [hunk 1]
```diff
@@ -177,23 +178,36 @@ extension ConfettiView {
         }
 
         // Colors
-        let purple = UIColor(red: 0.75, green: 0.35, blue: 0.95, alpha: 1.00)
-        let orange = UIColor(red: 1.00, green: 0.50, blue: 0.52, alpha: 1.00)
-        let green = UIColor(red: 0.44, green: 0.88, blue: 0.65, alpha: 1.00)
+        let purple = UIColor(light: UIColor(named: "Purple20") ?? .systemPurple,
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Notifications/Milestone Notifications/ConfettiView.swift]: You can use our Muriel colors here – check `MurielColor.swift` and `UIColor+MurielColors.swift`. I would suggest adding this convenience method to the top of `UIColor+MurielColors`:

```
class func muriel(name: MurielColorName, _ shade: MurielColorShade) -> UIColor {
    let newColor = MurielColor(name: name, shade: shade)
    return muriel(color: newColor)
}
```

and then you'll be able to reference these colours like: `.muriel(name: .purple, .shade20)`

**Review by frosty — COMMENTED** [review]
Looks and works great! I've left a suggestion about removing all the string literals for the colours.

File: WordPress/Classes/ViewRelated/Notifications/Milestone Notifications/ConfettiView.swift (hunk 1 above)
Comment (Gio2018) [reply WordPress/Classes/ViewRelated/Notifications/Milestone Notifications/ConfettiView.swift]: Done, thanks for the suggestion!

**Review by frosty — APPROVED** [review]
Changes look good! There's an outstanding question about swapping out the WordPressBlue for something else (as it's being deprecated), which we discussed on Slack. I think it's fine to either merge now and amend later, or tweak them in this PR and then merge.

**Comment by Gio2018** [comment]
> Changes look good! There's an outstanding question about swapping out the WordPressBlue for something else (as it's being deprecated), which we discussed on Slack. I think it's fine to either merge now and amend later, or tweak them in this PR and then merge.

I'm going ahead and merge this one, we can do another to change the colors when they are ready

**Comment by frosty** [comment]
Sounds good!

**Pull request merged**